examples/assets/scripts/utils/cubemap-renderer.js

```diff
@@ -0,0 +1,81 @@
+// How to use in the Editor:
+// - create 6 layers for rendering cubemap faces, and pass base name of these layers to .layerBaseName
+//   example: layerBaseName = "CubeLayer", layers "CubeLayer0", CubeLayer1" .. "CubeLayer5" need to be created in the Editor
+// - add your entities with Model component to these layers
+// - add this script component on an Entity, and use script assign .cubeMap to your material:
+//   material.cubeMap = entity.script.cubemapRenderer.cubeMap;
+
+var CubemapRenderer = pc.createScript('cubemapRenderer');
+
+CubemapRenderer.attributes.add('resolution', { type: 'number', default: 64 });
+CubemapRenderer.attributes.add('mipmaps', { type: 'boolean', default: false });
+CubemapRenderer.attributes.add('depth', { type: 'boolean', default: true });
+CubemapRenderer.attributes.add('layerBaseName', { type: 'string', default: "CubeLayer" });
+
+// initialize code called once per entity
+CubemapRenderer.prototype.initialize = function () {
+
+    // Create cubemap render target with specified resolution and mipmap generation
+    var colorBuffer = new pc.Texture(this.app.graphicsDevice, {
+        width: this.resolution,
+        height: this.resolution,
+        format: pc.PIXELFORMAT_RGBA8,
```

Comment:
This doesn't exist, does it? Don't you mean `PIXELFORMAT_R8_G8_B8_A8`?

Reply:
well spotted. That's what I get for copy & paste from internet .. perhaps used to exist?